cmd/start.go

```diff
@@ -28,6 +28,7 @@ func init() {
 	cmd.PersistentFlags().StringVar(&opts.Kubeconfig, "kubeconfig", opts.Kubeconfig, "Kubeconfig file to access a remote cluster (testing only)")
 	cmd.PersistentFlags().StringVar(&opts.NodeName, "node-name", opts.NodeName, "kubernetes node name CVO is scheduled on.")
 	cmd.PersistentFlags().BoolVar(&opts.EnableAutoUpdate, "enable-auto-update", opts.EnableAutoUpdate, "Enables the autoupdate controller.")
+	cmd.PersistentFlags().BoolVar(&opts.EnableDefaultClusterVersion, "enable-default-cluster-version", opts.EnableDefaultClusterVersion, "Allows the operator to create a ClusterVersion object if one does not already exist.")
```

Comment:
this makes the default `false`, when the previous behavior was `true`.  So that would be bad in terms of compatibility...

Reply:
> this makes the default `false`, when the previous behavior was `true`. So that would be bad in terms of compatibility...

Do we care what the default is if we always set this flag explicitly?

Reply:
I know only openshift uses this with explicit flags, so it doesn't matter that much. But I would prefer we don't make these incompatible changes.

I wouldn't block on it this time.

/lgtm